### DISCUSSION TITLE
What is the role of batch size for validation?

### DISCUSSION THREAD

**Discussion by nihil39** [ACEsuit/mace#694] [Q&A] [answered]
Hi, 

as per the tile, what is the role of the validation batch size if, form what I know, the weights and the gradient are not updated taking into account the validation set? Is it just a "memory trick" to make the whole validation set fit into memory? 
Sorry for the maybe trivial question.

Thank you.

**Comment by ilyes319** [comment, marked as answer]
Yes it is a memory trick to speed up the evaluation of the validation set, you can pick as high as your GPU memory allows.
